jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/component/Environment.java

```diff
@@ -17,14 +17,18 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
+import java.util.TreeMap;
 
 import org.eclipse.jetty.util.Attributes;
 import org.eclipse.jetty.util.TypeUtil;
 
+/**
+ * A named runtime environment containing a {@link ClassLoader} and {@link Attributes}.
+ */
 public interface Environment extends Attributes
 {
-    Environment CORE = ensure("core");
+    // Ensure there is a core environment for possible later deployments to it
+    Environment CORE = ensure("Core");
```

Comment:
All others are lower case and so they are in module files, so I'll keep it lower case.